duckduckgo/iOS
# Implements Welcome Links Navigation in Subscriptions

Task/Issue URL: https://app.asana.com/0/1200019156869587/1205017548855433/f

**Description**:
- Implements onAppear navigation for Settings, so you can navigate to a view programmatically when the view is shown
- Implements navigation from the Subscription Welcome page to each feature 



**Steps to test this PR**:

⚠️ Use a device + sandbox account

1.  Subscribe or restore a subscription
2. From the welcome page, tap any of the feature buttons 
3. Observe you are taken back to Settings and then to the selected feature. (or placeholder for it)
4. It should be enought to test one of the three buttons (VPN/ITP/DBP), but feel free to remove/restore the subscription and test all all them if you want.
5. Remove the subscription from the device and observe an ActionMessage (little "toast" message at the bottom) appears confirming it was removed



---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
